other/nightly/install.sh

```
@@ -27,6 +36,144 @@ ROOT_USERNAME=${ROOT_USERNAME:-}
+    fi
+}
+
+# Function to compare address pools
+compare_address_pools() {
+    local base1="$1"
+    local size1="$2"
+    local base2="$3"
+    local size2="$4"
+
+    # Normalize CIDR notation for comparison
+    local ip1=$(echo "$base1" | cut -d'/' -f1)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)